doc/espresso-dev-node.md

```diff
@@ -0,0 +1,71 @@
+| `rpc_url`                       | `Option<Url>`   | `ESPRESSO_SEQUENCER_L1_PROVIDER`     | Automatically launched Avil node if not provided.             | A JSON-RPC endpoint for the L1 to deploy to. If not provided, an Avil node will be launched automatically.                                   |
+| `mnemonic`                      | `String`        | `ESPRESSO_SEQUENCER_ETH_MNEMONIC`    | `test test test test test test test test test test test junk` | Mnemonic for an L1 wallet. This wallet is used to deploy the contracts, so the account indicated by `ACCOUNT_INDEX` must be funded with ETH. |
+| `account_index`                 | `u32`           | `ESPRESSO_DEPLOYER_ACCOUNT_INDEX`    | `0`                                                           | Account index in the L1 wallet generated by `MNEMONIC` to use when deploying the contracts.                                                  |
+| `sequencer_api_port`            | `u16`           | `ESPRESSO_SEQUENCER_API_PORT`        | Required                                                      | Port that the HTTP API will use.                                                                                                             |
+| `sequencer_api_max_connections` | `Option<usize>` | `ESPRESSO_SEQUENCER_MAX_CONNECTIONS` | None                                                          | Maximum concurrent connections allowed by the HTTP API server.                                                                               |
+| `builder_port`                  | `Option<u16>`   | `ESPRESSO_BUILDER_PORT`              | An unused port                                                | Port for connecting to the builder.                                                                                                          |
+| `prover_port`                   | `Option<u16>`   | `ESPRESSO_PROVER_PORT`               | An unused port                                                | Port for connecting to the prover. If this is not provided, an available port will be selected.                                              |
+| `dev_node_port`                 | `u16`           | `ESPRESSO_DEV_NODE_PORT`             | `20000`                                                       | Port for the dev node. This is used to provide tools and information to facilitate developers debugging.                                     |
+
+## APIs
+
+Once you have successfully run the dev node, you can access the corresponding ports to call the APIs of the
+[`builder`](https://docs.espressosys.com/sequencer/api-reference/builder-api),
+[`sequencer`](https://docs.espressosys.com/sequencer/api-reference/sequencer-api), and `prover`.
+
+In addition, you can access the `dev_node_port` to retrieve debugging information. Here are the details of the dev node
```

Comment:
Consider linking to the toml specification or generated docs (if we generate documentation from the specification).

Reply:
The toml specification is used to generate an HTML doc for users accessing to a wrong route/endpoint. Of course we can also link the toml file here, but just a bit weird. And `builder api` page doesn't link anything here, so I just followed the conventions.

Reply:
builder api documentation should also link to spec :smiley:  (IMO) .